django_short_urls/views.py

```diff
@@ -110,54 +111,72 @@ def main(request, path):
 
 
 @require_POST
-def new(request):
+def new(request):  # pylint: disable=too-many-branches
     '''
     Create a new short url based on the POST parameters
-    '''
 
-    if 'login' in request.REQUEST and 'api_key' in request.REQUEST:
-        login = request.REQUEST['login']
-        api_key = request.REQUEST['api_key']
+    Temporary pylint disable: OPS-4755 - once the temporary switches below have been removed,
+    remove the pylint disable above.
+    '''
+    user = None
+    if 'HTTP_AUTHORIZATION' in request.META:
```

Comment:
Will tackle in the followup as it makes the code a bit more complicated what with the fallback on url parameters for authentication.

https://work4labs.atlassian.net/browse/OPS-4805